core/domain/app_feedback_report_domain_test.py

```diff
@@ -211,6 +269,16 @@ def test_report_android_schema_version_not_an_int_validation_fails(
             'The report schema version %r is invalid, expected an integer' % (
                 self.android_report_obj.schema_version))
 
+    def test_report_ticket_id_none(
```

Comment:
Not sure what this test name means. Please format it based on the backend test guidance in the wiki.

Reply:
changed the test name `test_report_ticket_id_none`->`test_android_report_with_none_ticket_id_context_invalid_type_fails`